packages/core/src/utils/prepareEvent.ts

```diff
@@ -74,20 +76,25 @@ export function prepareEvent(
     result = finalScope.applyToEvent(prepared, hint);
   }
 
-  return result.then(evt => {
-    if (evt) {
-      // We apply the debug_meta field only after all event processors have ran, so that if any event processors modified
-      // file names (e.g.the RewriteFrames integration) the filename -> debug ID relationship isn't destroyed.
-      // This should not cause any PII issues, since we're only moving data that is already on the event and not adding
-      // any new data
-      applyDebugMeta(evt);
-    }
+  return result
+    .then(evt => {
+      // Process client-scoped event processors
+      return client && client.getEventProcessors ? notifyEventProcessors(client.getEventProcessors(), evt, hint) : evt;
```

Comment:
I actually noticed a problem with the implementation in one of the other PRs (for browser). Which is that we can't just call this after `prepareEvent` is completed in the baseclient, because then the stuff that happens in here below will not be applied correctly.

So I updated it to instead run this here. And I am wondering, what do you think is nicer:

1. Expose `getEventProcessors()` on the client (like I did here)
2. Instead expose e.g. `applyToEvent()` like we have on the scope and call that directly (which under the hood applies the event processors)?

cc @AbhiPrasad  / @Lms24  / @lforst

Reply:
Let's do `getEventProcessors` on the client - I'd rather not show more internals than what is needed.

Reply:
sounds good. we maybe will adjust this for v8 anyhow if we adjust the event processing pipeline, but should be good for now I guess.